### PR TITLE
Adding collar variation, fixed errors.

### PR DESCRIPTION
**Pull request by JonathanHue**
Added a variety of colors for collars and a rare chance for a spiked collar to be
spawned. Narky race was disabled however it created errors from the
commenting on it. So I removed it permanently. If it needs to be added
back, just look for an older version of race.dm and find the narky race
there, paste it in and bam, you got the narky race back.

I tried many methods to keep Narky's race in but have it disabled, every
time it generated errors. So I gave up in the end and removed it.

Kinkmate had an undefined object called handcuffs/kink, I commented it
out.

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -986,10 +986,11 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	vend_delay = 15
 	vend_reply = "Have fun, you shameless pervert!"
 	products = list(/obj/item/weapon/dildo=10,/obj/item/clothing/under/maid=5,/obj/item/clothing/under/schoolgirl=3,
-	/obj/item/clothing/tie/petcollar=5, /obj/item/clothing/ears/earmuffs=2,/obj/item/clothing/glasses/sunglasses/blindfold=2
+	/obj/item/clothing/tie/petcollar=5, /obj/item/clothing/ears/earmuffs=2,/obj/item/clothing/glasses/sunglasses/blindfold=2,
 	/obj/item/clothing/mask/muzzle=2)
 	contraband = list()
-	premium = list(/obj/item/weapon/dildo/psych=2,/obj/item/weapon/restraints/kink/=2,
```
Comment (Zeldazackman) [inline code/game/machinery/vending.dm]: /obj/item/weapon/restraints/handcuffs/kink

Comment (TalkingCactus) [inline code/game/machinery/vending.dm]: oops

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -989,8 +989,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	/obj/item/clothing/tie/petcollar=5, /obj/item/clothing/ears/earmuffs=2,/obj/item/clothing/glasses/sunglasses/blindfold=2,
 	/obj/item/clothing/mask/muzzle=2)
 	contraband = list()
-	premium = list(/obj/item/weapon/dildo/psych=2,
-	// /obj/item/weapon/restraints/kink/=2,
+	premium = list(/obj/item/weapon/dildo/psych=2,/obj/item/weapon/restraints/kink=2,
```
Comment (Zeldazackman) [inline code/game/machinery/vending.dm]: Its not going to find those cuffs as its "/obj/item/weapon/restraints/handcuffs/kink=2,".
For some reason its got the handcuffs as a parent rather then just being grouped with the rest of the restraints.

Comment (TalkingCactus) [inline code/game/machinery/vending.dm]: I fixed it in my PR

Comment (JonathanHue) [inline code/game/machinery/vending.dm]: Alright. Exclude my vending.dm

**Pull request closed without merging**